regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,13 +25,13 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
-        var gaview = new AnalyticsHandler(),
-            main = new MainView(),
-            sidebar = new SidebarView(),
-            header = new HeaderView(),  // Header before Drawer as Drawer sends Header events
-            drawer = new DrawerView();
+        var gaview = new AnalyticsHandler();
+        var header = new HeaderView();  // Header before Drawer as Drawer sends Header events
+        var drawer = new DrawerView();
+        var main = new MainView();
+        var sidebar = new SidebarView();
```

Comment:
That's just me trying to inject my js style preferences, although I borrowed them from airbnb: https://github.com/airbnb/javascript#variables. I think the frontend guild recommends the airbnb guide. Anyway, I'm happy to roll that back if you prefer.